### DISCUSSION TITLE
PHPのマルチバージョンの停止

### DISCUSSION THREAD

**Discussion by koriym** [alps-asd/app-state-diagram#122] [General]
Docker版が標準になればPHPは最新のバージョン１つだけサポートすれば良いかも？

**Comment by NaokiTsuchiya** [comment]
ApiDoc で利用しているので、単純に停止とはいかないかもしれないです
https://github.com/bearsunday/BEAR.ApiDoc/blob/1.x/composer.json#L26

**Reply by koriym** [reply]
そうでした！

**Reply by koriym** [reply]
Lockはあってもcloseはないんですねこれ。
